LICENSE

```diff
@@ -1,22 +1,121 @@
-Copyright (c) 2019, Mapbox
```

Comment:
License should stay the same. Just add a second line "Copyright (c) 2023, MapLibre contributors"